### DISCUSSION TITLE
Define a maximum aspect ratio

### DISCUSSION THREAD

**Discussion by MaluNoPeleke** [tailwindlabs/tailwindcss#5519] [Help]
### What version of @tailwindcss/aspect-ratio are you using?

v0.2.1

### What version of Node.js are you using?

v14

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://play.tailwindcss.com/OkIu0rEEMo

### Describe your issue

Is it possible to define a max aspect ratio that shouldn't be exceeded?
For example in the linked Playground you can see 21:9 image embedded with an aspect ratio of 16:9 and I would like to achieve that if an image exceeds the aspect ratios width it should simply display in full size instead of a fixed ratio.
So my reason to use this plugin is just to make sure that the height of an image is never >=0.5625 times as big as its width.

**Comment by reinink** [comment]
Hey there @MaluNoPeleke, I have moved this over to the Tailwind CSS discussions, as this isn't technically an issue with the `@tailwindcss/aspect-ratio` plugin. Hopefully you'll get some answers here. 👍
